airflow/lineage/datasets.py

```diff
@@ -104,13 +118,15 @@ def map_type(name):
 
 
 class DataBase(DataSet):
+    """DataBase class, subclass of DataSet."""
```

Comment:
This comment is almost worse than no comment at all. It doesn't tell me anything more about the class. If we aren't going to have a descriptibe doccomment that adds more info then I'd rather we just disabled the warning for these class